lib/domain-page/ensService.ts

```diff
@@ -101,13 +106,39 @@ export const getENSDomainDataThroughSubgraph = async (
   return data;
 };
 
+function getParent(domain: string): string {
```

Comment:
`getParent` sounded very big context for me, shall we call it `getOuterSubname`?

Please refer to [these specs](https://blockful.slack.com/archives/C03JQ5URTEG/p1722454458211099?thread_ts=1722437888.114199&cid=C03JQ5URTEG) before defining this new name

Reply:
@FrancoAguzzi It might be confusing? The purpose of this function is to return the domain's parent
<img width="404" alt="image" src="https://github.com/user-attachments/assets/895997b1-2431-4c7c-9c13-607ff262dfe9">

Reply:
@FrancoAguzzi done